### PR TITLE
Enable SCA exemption for stored/recurring payments

### PR DESCRIPTION
**Pull request by mjenssen**

### CODE REVIEW

File: src/elm/Service/Ticket.elm [hunk 1]
```diff
@@ -284,6 +284,9 @@ encodePaymentSelection paymentSelection =
 isScaExemption : PaymentSelection -> Bool
 isScaExemption paymentSelection =
     case paymentSelection of
+        Recurring _ ->
```
Comment (mikaelbr) [inline src/elm/Service/Ticket.elm:287]: Yeah, because it only makes sense on recurring, not even NETS when not recurring?

**Review by mikaelbr — APPROVED** [review]

File: src/elm/Service/Ticket.elm (hunk 1 above)
Comment (mjenssen) [reply src/elm/Service/Ticket.elm:287]: I'm pretty sure for non-recurring Nets payments, SCA will be done in the same flow. I suppose we can try to get an exemption though. What do you think?

Comment (gorandalum) [reply src/elm/Service/Ticket.elm:287]: You have probably already taken this into consideration, but when sca was enabled earlier it was enabled for everything except Vipps.

```elm
case paymentSelection of
  NonRecurring PaymentType.Vipps ->
    False

  _ ->
    True
```

Comment (mjenssen) [reply src/elm/Service/Ticket.elm:287]: Yes, we had a short discussion in the office and decided to go with enabling SCA exemption for all Nets payments.

**Pull request merged**